docs/_index.md

```diff
@@ -122,10 +125,10 @@ Viam supports a wide variety of systems, including:
         <h4>Configure your robots</h4>
         <div style="text-align: left">
             <ol style="padding-inline-start: 1.1rem">
-            <li><a href="/manage/fleet/robots/">Set up your robot</a> or<a href="manage/fleet/"> fleet</a></li>
             <li><a href="installation/">Install Viam on your robot</a></li>
-            <li><a href="manage/configuration/">Configure your robot</a></li>
-            <li><a href="manage/fleet/robots/#control">Test your robot</a></li>
+            <li><a href="manage/configuration/">Configure your robot<a> or <a href="manage/fleet/">fleet</a></li>
```

Comment:
This ordering doesn't make sense. Configuring the fleet happens before installing viam.